drivers/serial/Kconfig.nrf5

```diff
@@ -32,23 +32,26 @@ config UART_NRF5_FLOW_CONTROL
 
 config UART_NRF5_GPIO_TX_PIN
 	int "TX Pin Number"
-	range 0 31
+	range 0 31 if CONFIG_SOC_NRF52832_QFAA
```

Comment:
No need of if clause, old one will cover nrf52832 and nrf51

Have the
Range 0 47 if ... 
above the old range 0 31

Reply:
OK. Sorry, I'm not very familiar with the KConfig syntax